backend/src/controllers/ratings.controller.ts

```diff
@@ -0,0 +1,54 @@
+import { NextFunction, Request, Response } from 'express';
+import { CreateRatingsDto } from '@dtos/ratings.dto';
+import { Ratings } from '@interfaces/ratings.interface';
+import ratingService from '@services/ratings.service';
+
+class RatingController {
```

Comment:
Maybe add endpoint and controller function to get a rating for a product combined.
=> 3,4 Stars with 34 ratings

Reply:
[f84a1d4]